index.html

```diff
@@ -3078,25 +3078,62 @@ <h3>Extensibility</h3>
         <section>
           <h4>Semantic Interoperability</h4>
 
+          <p>
+            When defining new terms in an application-specific vocabulary, developers MUST use globally unambiguous [=URLs=] to identify the terms.
+            Whenever possible, it is RECOMMENDED to re-use terms — and their corresponding URLs — defined by well-known, public vocabularies,
+            such as [[[?DCTERMS]]] [[?DCTERMS]] or [[[?schema-org]]] [[?schema-org]]. 
+            If that is not possible, developers MUST <em>define</em> a new URL for each term. 
+            When doing so, general [[LINKED-DATA]] principles must be followed, in particular:
+          </p>
+
           <ul>
-            <li>
-JSON-LD-based processors MUST produce an error when a JSON-LD context redefines
-any term in the
-<a href="https://www.w3.org/TR/json-ld/#dfn-active-context">active context</a>.
-The only way to change the definition of existing terms is to introduce a new
-term that clears the active context within the scope of that new term. Authors
-that are interested in this feature should read about the
-`@protected` feature in the JSON-LD 1.1 specification.
-            </li>
-          </ul>
+              <li>
+                Human-readable documentation MUST be published, describing the semantics of and
+                the constraints on the use of each term.
+              </li>
+              <li>
+                  It is RECOMMENDED to also publish the collection of all new terms as a machine-readable vocabulary 
+                  using [[[?RDF-SCHEMA]]] [[?RDF-SCHEMA]] (and, if necessary, OWL2 [[?OWL2-OVERVIEW]]).
+              </li>
+              <li>
+                  It MUST be possible to dereference the URL of a term, resulting in its description and/or formal definition.   
+              </li> 
+          </ul>       
+             
+          <p>
+            Developers SHOULD follow the <a data-cite="?LD-BP#vocabulary-checklist">detailed checklist</a> in
+            [[[?LD-BP]]] [[?LD-BP]] when defining a new vocabulary.
+          </p>
+          <p>
+              Furthermore, a machine-readable description (that is, a 
+              <a data-cite="JSON-LD11#dfn-context">JSON-LD Context document</a>) MUST be published at the URL specified
+              in the `@context` [=property=] for the vocabulary. 
```

Comment:
We don't need to say this. The system will just not work if a context isn't published at that location.

Reply:
I am not sure I understand this remark. Yes, the document will be non conformant, because the JSON-LD processor will fail if the `@context` value does not dereference. But if the user does not publish a context document _nor_ does he/she put a `@context` property for the vocabulary, the document will pass, thanks to the `@vocab` in the core context...